### PR TITLE
Also store success boolean to pickle output when using multiprocessing

### PR DESCRIPTION
**Pull request by philippotto**
### Description:
- When using the clustertools with slurm, the result pickle file was always written in the format of `(wasSuccessful, result_value)`. However, when using multiprocessing, this was not the case which was inconsistent.

### Todos:
Make sure to delete unnecessary points or to check all before merging:
 - [x] Updated Changelog

### CODE REVIEW

File: cluster_tools/cluster_tools/remote.py [hunk 1]
```diff
@@ -78,10 +78,10 @@ def worker(executor, workerid, job_array_index, job_array_index_offset, cfut_dir
         out = pickling.dumps(result)
 
     except Exception:
-        print(traceback.format_exc())
 
         result = False, format_remote_exc()
-        logging.warning("Job computation failed.")
+        logging.warning("Job computation failed with:")
+        print(traceback.format_exc())
```
Comment (jstriebel) [inline cluster_tools/cluster_tools/remote.py:84]: I'm wondering why this is partially in logging.warning and in print? Maybe we should rather put the traceback also in the logging statement?

File: cluster_tools/cluster_tools/__init__.py [hunk 2]
```diff
@@ -162,7 +162,12 @@ def _execute_and_persist_function(output_pickle_path, *args, **kwargs):
         func = args[0]
         args = args[1:]
 
-        result = func(*args, **kwargs)
+        try:
+            result = True, func(*args, **kwargs)
+        except Exception as exc:
+            result = False, exc
+            logging.warning("Job computation failed with:")
+            print(exc)
```
Comment (jstriebel) [inline cluster_tools/cluster_tools/__init__.py:170]: Same here, maybe also in other spots?

**Review by jstriebel — APPROVED** [review]
Nice, mostly LGTM :+1: Just a question about the logging:

File: cluster_tools/cluster_tools/remote.py (hunk 1 above)
Comment (philippotto) [reply cluster_tools/cluster_tools/remote.py]: good point. i cleared it up now :ok_hand:

**Pull request merged**